tools/generate-legioncomponent-template.js

```diff
@@ -0,0 +1,58 @@
+require('colors');
```

Comment:
do we need separate scripts for this? Can't we just pass in a path or something to teh existing script? and/or use a different template but use the same script?

Reply:
I would rather extend the script to take an arg for AppName or something and then it can put the componets in the right folder based on that name.

Reply:
I like that suggestion, I'll create one that takes AppName.